content/docs/reference/pkg/_index.md

```diff
@@ -11,18 +12,94 @@ as higher-level APIs that make it easier to deliver cloud applications and
 infrastructure.
 
 These APIs are available as packages in your chosen language's package manager
-&mdash; NPM for Node.js and PyPI for Python. There is a dedicated package for
+&mdash; npm for TypeScript/JavaScript and PyPI for Python. There is a dedicated package for
 each cloud that includes access to its full capabilities. In addition, Pulumi
 offers many convenience packages that make common tasks easier, like setting
 up a network, creating a Kubernetes cluster, and building and publishing containers
 to private registries.
 
-These packages can be mixed to enable multi-cloud and a spectrum of control to productivity:
+### Choose Your Language
 
-![Pulumi Library Architecture](/images/docs/reference/pkg-arch-layers.png)
+{{< langchoose nogo >}}
 
-You can read further documentation specific to each language here:
+### General Purpose Packages
 
-* [JavaScript/TypeScript](nodejs)
-* [Python](python)
-* Go (coming soon!)
+The Pulumi SDK package is used for accessing the core programming model around resources, configuration, etc.
```

Comment:
I don't like using etc. in technical documentation, so how about "configuration, and other components"?